### PR TITLE
implement selection page

### LINKED ISSUE #80: Frontend: Selection Page
When I am registering and login in for the first time, I want to choose between a student, mentor or staff so I can use the web app accordingly to my role.

Create a page below that considers the data retrieved with the endpoint developed on the issue to show the list #58 and the page component created on this issue #61 - SelectionPage - src/client/containers/SelectionPage/SelectionPage.component.js

image

IMPORTANT: make sure you use existing components, if needed change the existing components or work together with the developer that created those same components.

### PR DESCRIPTION
**Pull request by claudion3**
# Description

This PR When is the page you get after you registering and login in for the first time and you have option to choose between a student, mentor or staff 

Fixes #80 

# How to test?
seletionPage

# Checklist
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/SelectionPage/SelectionPage.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "./Selectionpage.css";
+import Radio from '../Radio/ButtonRadio'
+import HYFLogo from './HYF.logo.svg';
+
+const SelectOption=()=>{
```
Comment (tpetrina) [inline src/client/components/SelectionPage/SelectionPage.js:6]: How did you manage to push this, formatting is way off in this file.

File: src/client/components/SelectionPage/Selectionpage.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.seletionPage {
```
Comment (tpetrina) [inline src/client/components/SelectionPage/Selectionpage.css:1]: Typo: it should be `.selection-page` - missing `c`.

File: src/client/components/SelectionPage/Selectionpage.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  width: 30%;
+  margin-left: 33.3%;
+  padding: 1% 2%;
+}
+.text {
+  margin: 5% 3%;
+  padding: 1% 2%;
+  color: rgb(165, 178, 180);
+}
+/*Responsive */
```
Comment (tpetrina) [inline src/client/components/SelectionPage/Selectionpage.css:27]: Remove this comment and replace with blank line.

File: src/client/components/SelectionPage/Selectionpage.stories.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SelectOption from './SelectionPage';
+
+// import { action } from '@storybook/addon-actions';
```
Comment (tpetrina) [inline src/client/components/SelectionPage/Selectionpage.stories.js:4]: Remove action if not used.

File: src/client/components/SelectionPage/Selectionpage.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
+.seletionPage {
+  width: 100%;
+  text-align: center;
+}
+.logo {
```
Comment (tpetrina) [inline src/client/components/SelectionPage/Selectionpage.css:5]: All other selectors in this file should start with `.selection-page` to not clash with other styles on other pages.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Formatting and typos.

File: src/client/components/SelectionPage/SelectionPage.js (hunk 1 above)
Comment (claudion3) [reply src/client/components/SelectionPage/SelectionPage.js:6]: i took it from README.md because i couldn't find that logo in other format

**Review by tpetrina — APPROVED** [review]
Good work!

**Review by MaherHussain — APPROVED** [review]

**Pull request merged**